cargo-dist/src/errors.rs

```diff
@@ -229,6 +229,13 @@ pub enum DistError {
         /// The missing keys
         keys: &'static [&'static str],
     },
+    /// unrecognized style
+    #[error("{style} is not a recognized value")]
+    #[diagnostic(help("Styles that do not come with cargo-dist should be prefixed with an @"))]
```

Comment:
error message now out of date, also "Styles" is too vague here. publish-jobs or jobs or... something more precise